Figures/Helper.cs

```diff
@@ -0,0 +1,271 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualBasic.CompilerServices;
+
+namespace FiguresProject
+{
+    internal class Helper
+    {
+        public static void PrintMenu()
+        {
+            Console.WriteLine("1. Add new figure");
+            Console.WriteLine("2. Change Figure parameters");
+            Console.WriteLine("3. Remove figure");
+            Console.WriteLine("4. Draw the figure");
+            Console.WriteLine("5. Exit");
+        }
+
+        public static void AddFigure(List<Figures> figureList)
```

Comment:
The function is very messy and long. Could be divided into more deliberate functions.